.github/workflows/build.yml

```
@@ -0,0 +1,57 @@
+name: Build Website
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+      - dev
+
+concurrency:
+  group: pl-website-build-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  AWS_REGION: us-east-1
+
+jobs:
+  build:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.7
+
+      - name: Install Dependencies
+        run: |
+          pip install -r requirements.txt
+
+      - name: Build
+        run: |
+          mkdir -p site/carnival
+          mkdir -p site/codecamp
+          python compile.py
+
+      - name: Upload Pull Request Artifact
+        uses: actions/upload-artifact@v3
+        with:
+          name: site
+          path: site
+          retention-days: ${{ github.event_name == 'pull_request' && 10 || 30 }}
+
+      - name: Push changes to dev (pennylane-ai-website-website-non-react-dev)
```

```suggestion
      - name: Push changes to dev (pennylane-ai-website-non-react-dev)
```
not sure if it's a typo? also repeated below if that's the case

---

:sweat_smile: ... I accidentally added -website- twice to the S3 bucket name .. this is the name of S3 bucket verbatim ... I can change it but changing project name will require re-deploy of all infra

---

Now that I'm thinking about it, I can potentially create a new bucket without the repeat and do an S3 sync :thinking: ... let me fix this one sec

---

Bucket names updated :+1: